vitest.config.ts

```diff
@@ -0,0 +1,8 @@
+/// <reference types="vitest" />
+import { getViteConfig } from 'astro/config';
+
+export default getViteConfig({
+  test: {
+    // Vitest configuration options
```

Comment:
Would be nice to add the [Github Actions reporter](https://vitest.dev/guide/reporters#github-actions-reporter) so our code is annotated like this:

![Code annotated with test feedback](https://github.com/vitest-dev/vitest/assets/4232207/ce8447c1-0eab-4fe1-abef-d0d322290dca)